src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/DocumentFormattingTest.cs

```diff
@@ -2561,8 +3232,8 @@ @model BlazorApp58.Pages.Index2Model
                     </section>
                     <section class="section">
                         <div class="container">
-                            @foreach (var item in Model.Images)
-                            {
+                    @foreach (var item in Model.Images)
+                    {
```

Comment:
This change is due to the document being syntactically invalid. The Razor parser doesn't see the close brace as a C# token, and the section and div elements have start and end tags that don't match.